### DISCUSSION TITLE
Conflect happens when using 'id' in notification

### DISCUSSION THREAD

**Discussion by chuja3** [spatie/laravel-uptime-monitor#326] [Q&A]
When passing data to the Notification class, with an "id" name, the new notification's id will take the value of the id passed in the construct method, and it will give a duplicate entry error when creating a second notification.
this happened only with "id", if you change the name to itemId or orderId, it will work fine.

**Comment by freekmurze** [comment]
Feel free to send a PR that fixes this. Do include tests.
